README.md

```diff
@@ -1,63 +1,55 @@
 # mros_reasoner
+
 A meta-controller implementation for ROS1
 
 ## Installation
 
 ### Previous steps
 
-- First you need to have a ROS Melodic version with Python3 support, for example following these [instructions](https://answers.ros.org/question/237613/how-to-define-ros-kinetic-to-use-python3-instead-of-python27/?answer=331009#post-id-331009)
-_Notes:_
-  - For me it worked with `Melodic` and `python3.6-venv`
-  - I created `$HOME/rospy3_melodic` for the ROS Melodic version with Python3 support
-
-- **Note:** For regular usage you'll need to make sure to have the virtual environment activated, or things will break.
+- you need to install a `java jre` you can install it with:
```

Comment:
Suggestion: add a short comment on *why* users need this.

Reply:
I've made an explanation about this on https://github.com/tud-cor/mc_mros_reasoner/commit/7e98fa74e7182e6883019db47858b6d752bef148